src/components/MedicationReports.js

```diff
@@ -30,10 +30,17 @@ const MedicationReports = observer(({ assembly }) => (
       .participant_account
       .information
       .medication_reports
+      .sort(function(a,b){
+        return new Date(b.timestamp) - new Date(a.timestamp);
```

Comment:
I tried looking through the documentation, but is there a way to use `DateTime.diff` here?

Reply:
You can just subtract new DateTime() ! Good catch this provides consistency throughout the app.